pkg/system/phase4_configuring.go

```diff
@@ -409,8 +407,10 @@ func (r *Reconciler) setDesiredEndpointMounts(podSpec *corev1.PodSpec, container
 	container.VolumeMounts = r.DefaultDeploymentEndpoint.Containers[0].VolumeMounts
 
 	for _, nsStore := range namespaceStoreList.Items {
-		if nsStore.Status.Phase == nbv1.NamespaceStorePhaseRejected {
-			log.Warnf("namespacestore %s is in rejected phase", nsStore.Name)
```

Comment:
maybe leave this warning next to the continue - just change to namespace store is not valid. WDYT?

Reply:
You told me that you don't want to see this warning each 3 sec, no?

Reply:
After our small discussion - it's fine we will get the reject reason from the namespacestore describe